src/test/java/plugins/StageViewTest.java

```diff
@@ -0,0 +1,96 @@
+package plugins;
+
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.po.Build;
+import org.jenkinsci.test.acceptance.po.BuildHistory;
+import org.jenkinsci.test.acceptance.po.WorkflowJob;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsString;
+
+/**
+ * Created by boris on 17.04.17.
+ * Base Implementation of the stageview test.
+ * TODO: Remove inline editor Tests and store in files/improve with builder pattern
+ * TODO: Build po for
+ */
+public class StageViewTest extends AbstractJUnitTest{
+
+    /**
+     * This tests create a simple stage. It checks if after the first build the stage view is now part of the job page.
+     * @throws Exception
+     */
+    @WithPlugins("workflow-aggregator@1.1")
+    @Test
+    public void jobShouldContainStageview() throws Exception {
```

Comment:
Einige generelle Anmerkungen zum Style Guide:
- Annotationen am besten in eine Zeile, das wichtigste zuerst: @Test @WithPlugins 
- Versionsnummer am besten Weglassen, dann ist das die latest 
- Annotation WithPlugins kann auch an der Klasse stehen
- Aus historischen Gründen nutzen wir underscores statt CamelCase
- throws Exception kann weg, oder wer wirft eine Exception?